### DISCUSSION TITLE
doc: Example workflow demonstrating 'since' or 'baseline' feature?

### DISCUSSION THREAD

**Discussion by draptik** [stryker-mutator/stryker-net#3127] [Q&A]
Could somebody provide a simple git workflow demonstrating the usage of 'since' and/or 'baseline'?

**Comment by rouke-broersma** [comment]
Hi, did you see https://stryker-mutator.io/docs/stryker-net/stryker-in-pipeline/#configuring-baseline-in-pull-requests ?

**Comment by draptik** [comment]
@rouke-broersma thanks for the pointer!

The provided link appears to apply `Choose a storage provider (Dashboard for public projects or Azure File Share for private projects)`, but not to local Git, is this correct?

Is there an offline way of using the `since` and `baseline` feature?

**Reply by rouke-broersma** [reply]
Yes, the default provider is 'Disk'. This requires no setup, it works as soon as you turn it on. What further information would you like on it?

**Reply by draptik** [reply]
An example would help - otherwise I wouldn't be asking ;-)
I have tried both without success.
How to use a git-hash?

**Reply by rouke-broersma** [reply]
`dotnet stryker --with-baseline:development`

If you let us know what you have tried so far we may be able to help you figure out what isn't working for you.

**Reply by draptik** [reply]
Thanks for getting back to me: I'll try to write a reproducible script next weekend.

**Comment by draptik** [comment]
I've created a minimal project describing my problem:

https://github.com/draptik/stryker-since-demo

I have a project and create a git tag (i.e. `v1`).

My assumption is that the Stryker magic ignores all findings before that tag when included in the config like

```json
    "since": {
      "enabled": true,
      "ignore-changes-in": [],
      "target": "v1"
    },
```

But that assumption seems to be wrong. ;-)

How should one use this feature?
